README.md

```diff
@@ -29,7 +31,7 @@ The guidelines can be found here (TBD)
 
 ## Trade off
```

Comment:
Might as well change this to 'Trade-off' or 'Tradeoff'

Reply:
ok

Reply:
ok